bycycle/features/features.py

```diff
@@ -0,0 +1,146 @@
+        - 'trough' : cycles are defined peak-to-peak
+
+    burst_detection_method : string, optional, default: 'cycles'
+        Method for detecting bursts.
+
+        - 'cycles': detect bursts based on the consistency of consecutive periods & amplitudes
+        - 'amplitude': detect bursts using an amplitude threshold
+
+    burst_detection_kwargs : dict, optional, default: None
+        Keyword arguments defined in either:
+
+        - :func:`~.detect_bursts_cycles` for consistency burst detection
+          (i.e. when burst_detection_method == 'cycles')
+        - :func:`~.detect_bursts_df_amp` and :func:`~.compute_burst_fraction` for dual
+          amplitude threshold burst detection (i.e. when burst_detection_method == 'amplitude').
+
+    find_extrema_kwargs : dict, optional, default: None
+        Keyword arguments for function to find peaks an troughs (:func:`~.find_extrema`)
+        to change filter Parameters or boundary. By default, it sets the filter length to three
+        cycles of the low cutoff frequency (``f_range[0]``).
+    hilbert_increase_n : bool, optional, default: False
+        Corresponding kwarg for :func:`~neurodsp.timefrequency.hilbert.amp_by_time` for determining
+        ``band_amp``. If true, this zero-pads the signal when computing the Fourier transform, which
+        can be necessary for computing it in a reasonable amount of time.
+    return_samples : bool, optional, default: True
+        Returns samples indices of cyclepoints used for determining features if True.
+
+    Returns
+    -------
+    df_features : pandas.DataFrame
+        A dataframe containing shape and burst features for each cycle. Columns:
+
+        - ``period`` : period of the cycle
+        - ``time_decay`` : time between peak and next trough
+        - ``time_rise`` : time between peak and previous trough
+        - ``time_peak`` : time between rise and decay zero-crosses
+        - ``time_trough`` : duration of previous trough estimated by zero-crossings
+        - ``volt_decay`` : voltage change between peak and next trough
+        - ``volt_rise`` : voltage change between peak and previous trough
+        - ``volt_amp`` : average of rise and decay voltage
+        - ``volt_peak`` : voltage at the peak
+        - ``volt_trough`` : voltage at the last trough
+        - ``time_rdsym`` : fraction of cycle in the rise period
+        - ``time_ptsym`` : fraction of cycle in the peak period
+        - ``band_amp`` : average analytic amplitude of the oscillation computed using narrowband
+          filtering and the Hilbert transform. Filter length is 3 cycles of the low cutoff
+          frequency. Average taken across all time points in the cycle.
+
+        When consistency burst detection is used (i.e. burst_detection_method == 'cycles'):
+
+        - ``amplitude_fraction`` : normalized amplitude
+        - ``amplitude_consistency`` : difference in the rise and decay voltage within a cycle
```

Comment:
I think at some point `amp` got expanded in `amplitude` for feature names? I'm not sure we need this expansion - it makes those labels a fair bit longer, and for this case I think `amp` is a pretty clear short version.

Reply:
Renamed as you suggested (a5f4ed8).